IOC-AOP-SpringBoot/homework1/src/main/java/ru/otus/homework/utils/CSVParser.java

```diff
@@ -0,0 +1,44 @@
+package ru.otus.homework.utils;
+
+import au.com.bytecode.opencsv.CSVReader;
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.io.Resource;
+import ru.otus.homework.dao.QuestionDao;
+import ru.otus.homework.domain.Option;
+import ru.otus.homework.domain.Question;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+@RequiredArgsConstructor
+public class CSVParser {
+    private final Resource resource;
+
+    private final QuestionDao questionDao;
+
+    public void parse() {
+        try (CSVReader reader = new CSVReader(new FileReader(resource.getFile()), ';')) {
```

Comment:
Если починить создание jar и запустить его, то будет:
>java.io.FileNotFoundException: class path resource [questions.csv] cannot be resolved to absolute file path because it does not reside in the file system:

Это происходит т.к. не выполнен пункт:
>5. CSV с вопросами читается именно как ресурс, а не как файл.